client/app/_components/ui/logo/Logo.tsx

```diff
@@ -0,0 +1,24 @@
+import Image from 'next/image'
+
+const Logo = ({ width = 150, height = 150 }) => {
```

Comment:
Hmm I'm not sure if we need the component for Logo though! We only use Logo for the header, and we can just put Logo inside of the header. I feel this is a too small component 👀  Maybe you won't get the warning if you do so.

Reply:
I deleted the logo component since it's used just for header!
And I figured out the way to get rid of the warning by adding some styles!
https://github.com/HAK2024/fairy-share/pull/55/commits/160cbf1849f279bfd78810aebcf0a22d6f3435bb